container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -3979,6 +4023,10 @@ variables:
   options:
     default: 'splatform/eirini-cert-copier:1.0.0.4.gd8e7208'
     description: "The docker image used by Eirini to register the image registry CA cert with Docker, on each Kubernetes node"
+- name: EIRINI_EIRINI_EXTENSIONS_SERVICE_HOST
+  options:
+    type: environment
+    description: "The Eirini extensions service ClusterIP"
```

Comment:
Please mention that it's automatically set by Kubernetes

Reply:
Should be fixed in: https://github.com/SUSE/scf/pull/2397/commits/e2aca801407febacf5d0174fcc04aa672eb1ca47